pom.xml

```diff
@@ -83,7 +83,7 @@
     <maven.compiler.source>${java.version}</maven.compiler.source>
     <maven.compiler.target>${java.version}</maven.compiler.target>
     <spotless-maven-plugin.version>2.30.0</spotless-maven-plugin.version>
-    <metrics.version>3.1.0</metrics.version>
+    <metrics.version>3.1.2</metrics.version>
```

Comment:
Can you explain why upgrade metrics.version, is this necessary?

Reply:
[WARNING] Rule 0: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for io.dropwizard.metrics:metrics-core:3.1.0 paths to dependency are:
+-org.apache.uniffle:coordinator:0.10.0-SNAPSHOT
  +-org.apache.uniffle:rss-common:0.10.0-SNAPSHOT
    +-io.prometheus:simpleclient_dropwizard:0.8.0
      +-io.dropwizard.metrics:metrics-core:3.1.0 (managed) <-- io.dropwizard.metrics:metrics-core:3.1.2

Reply:
As you change to use `<prometheus.simpleclient.version>0.9.0</prometheus.simpleclient.version>` for the version of `io.prometheus:simpleclient_dropwizard` now, can you try to build again without `metrics.version` change and check whether the enforcing error exists? If not, you can paste the new log part here again.

Reply:
> @kuszz As you change to use `<prometheus.simpleclient.version>0.9.0</prometheus.simpleclient.version>` for the version of `io.prometheus:simpleclient_dropwizard` now, can you try to build again without `metrics.version` change and check whether the enforcing error exists? If not, you can paste the new log part here again.
@maobaolong 
[INFO] --- enforcer:1.4.1:enforce (enforce) @ coordinator ---
[WARNING] Rule 0: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for io.dropwizard.metrics:metrics-core:3.1.0 paths to dependency are:
+-org.apache.uniffle:coordinator:0.10.0-SNAPSHOT
  +-org.apache.uniffle:rss-common:0.10.0-SNAPSHOT
    +-io.prometheus:simpleclient_dropwizard:0.9.0
      +-io.dropwizard.metrics:metrics-core:3.1.0 (managed) <-- io.dropwizard.metrics:metrics-core:3.1.2

Reply:
Ok, it's clear for now